elastic4s-http-streams/src/main/scala/com/sksamuel/elastic4s/streams/ScrollPublisher.scala

```diff
@@ -150,7 +150,7 @@ class PublishActor(client: HttpClient,
     // more results and we can unleash the beast (stashed requests) and switch back to ready mode
     case Success(resp: SearchResponse) =>
       scrollId = resp.scrollId.getOrElse("Query didn't return a scroll id")
-      queue.enqueue(resp.hits.hits: _*)
+      resp.hits.hits.foreach(h => queue.enqueue(h))
```

Comment:
could be niced with `queue.enqueueAll` and `scala-collection-compat` lib.
I tried it but its missing support for `mutable.Queue.enqueueAll`.
I added it in a PR to the compat lib, but might take a couple of weeks until it appears in a release.